### ISSUE TITLE
In version 7.5.6, and now in 7.7, File Open/Search feature is not present/does not work anymore.

### ISSUE THREAD

**Issue by smp0806** [notepad-plus-plus/notepad-plus-plus#5766]
### Description of the Issue
In version 7.5.6, and now in latest version 7.7, in Menu File\Open the Search field feature in the top right hand side, is not present/does not work anymore.
The PC is on Windows 7 Enterprise Sp1 English 64-bit, Region and Language: English US.
### Steps to Reproduce the Issue

1.  in Menu File
2. Open 
3. The Search field in the top right hand side, is not present/does not work anymore. 

### Expected Behavior
Work well in version version 7.3.3 for example

### Debug Information
I am already back in version 7.3.3 with such Debug info:
Notepad++ v7.3.3   (32-bit)
Build time : Mar  8 2017 - 03:37:37
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 7 (64-bit)
Plugins : mimeTools.dll NppConverter.dll NppExport.dll NppFTP.dll PluginManager.dll 

As a work around on both versions noticed,  I have to downgrade back to version 7.3.3 to see the feature working again.

Thank you in advance to fix this bug, please? 
Best regards
Stephane

See attached file: NotepadPlusPLus_7_5_6_7_7_Menu_File_Open_Search_DoesntNotWork.jpg for visual understanding.

**Comment by smp0806** [comment]
![NotepadPlusPLus_7_5_6_7_7_Menu_File_Open_Search_DoesntNotWork](https://user-images.githubusercontent.com/51603656/59182563-fcbf6100-8b61-11e9-9527-f343b798fcf6.jpg)

**Comment by MetaChuh** [comment]
@smp0806 

i can not reproduce a missing search field at the new style dialog `file > open`menu, using notepad++ 7.7 on windows 7 x64 and windows 10 x64 (see screenshot below)

<img width="957" alt="" src="https://user-images.githubusercontent.com/45773305/59189350-284e4580-8b7a-11e9-80ac-cd9c415fec4d.png">

eventually `new style dialog` might be disabled at your system.
please go to the menu `settings > preferences > default directory`, and make sure that `new style dialog` is active, as seen at the screenshot below.

![Imgur](https://i.imgur.com/fibS0kv.png)

best regards.

-----
```
general notes:

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
(no extra account is needed, just use your github account to sign in)
```
[<p align="center"><kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd></p>](https://notepad-plus-plus.org/community/)

**Comment by smp0806** [comment]
Hi,
The following setting fixed the issue in the version 7.7:
eventually new style dialog might be disabled at your system.
please go to the menu settings > preferences > default directory, and make sure that new style dialog is active, as seen at the screenshot below.

Thank you very much indeed
Best regards


Mr Stéphane MESONERO-PEREZ
stephanemesoneroperez@yahoo.fr

     On Monday, 10 June 2019, 11:28:00 BST, MetaChuh <notifications@github.com> wrote:  
 
 
@smp0806

i can not reproduce a missing search field at the new style dialog file > openmenu, using notepad++ 7.7 on windows 7 x64 and windows 10 x64 (see screenshot below)



eventually new style dialog might be disabled at your system.
please go to the menu settings > preferences > default directory, and make sure that new style dialog is active, as seen at the screenshot below.



best regards.
general notes:for general questions, or if it is not sure, whether your issue is directly related tothe notepad++ source code, please visit us at the notepad++ community forum, andsearch if related topics exists. you are welcome to post either at similartopics, or to create a new topic at Help Wanted or General Discussion.(no extra account is needed, just use your github account to sign in)


                         Click here to visit the Notepad++ Community Forum
